packages/host/app/lib/matrix-classes/message-builder.ts

```diff
@@ -193,7 +195,7 @@ export default class MessageBuilder {
         command.commandRequest = decodeCommandRequest(encodedCommandRequest);
       } else {
         message.commands.push(
-          this.buildMessageCommand(
+          await this.buildMessageCommand(
```

Comment:
is it desired that this be performed serially rather than in parallel (via Promise.all)?

Reply:
I think serialially is fine. The only async op is the loader.import and I expect the module to be loaded already in the majority of cases.